recipe/meta.yaml

```diff
@@ -8,22 +8,24 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eth_abi-{{ version }}.tar.gz
-  sha256: c3872e3ac1e9ef3f8c6599aaca4ee536d536eefca63a6892ab937f0560edb656
+  sha256: 33ddd756206e90f7ddff1330cc8cac4aa411a824fe779314a0a52abea2c8fc14
 
 build:
+  skip: True  # [py<39]
```

Comment:
As a nit on this `setup.py` says `>=3.8` so we should skip `<py38`.  This is so that anyone using our recipe gets a true reflection of the upstream, not our choices regarding what we support.

Reply:
Skip for py38 is intentional, as we don't have `eth-utils` for py38. I'll add such comment to recipe.